packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -4146,15 +4186,42 @@ class _ScribbleFocusableState extends State<_ScribbleFocusable> implements Scrib
 
   @override
   void onScribbleFocus(Offset offset) {
-    widget.focusNode.requestFocus();
-    renderEditable?.selectPositionAt(from: offset, cause: SelectionChangedCause.scribble);
-    widget.updateSelectionRects();
+    return widget.onScribbleFocus(offset);
+  }
+
+  @override
+  void insertTextPlaceholder(Size size) {
+    if (!widget.enabled) {
```

Comment:
Nit: can these two if statements be merged into one?

Reply:
Should `readOnly` also be checked here?

Reply:
For your nit, yes, done.

About readOnly, CC @fbcouch who wrote this originally.  Should this return if readOnly is true?

I just moved this code, but if it should check readOnly then maybe we can fix it here or in another PR.

Reply:
That sounds right to me – I don't think there's any reason to let users insert a placeholder (long press with the pencil to open up space to write) on a readOnly EditableText...I probably just didn't realize that we had both readOnly and enabled

Reply:
Done!  Thanks for the quick response.  I'll double check this on a real device later.